fdbkubernetesmonitor/main.go

```diff
@@ -103,6 +105,8 @@ func main() {
 	pflag.BoolVar(&enablePprof, "enable-pprof", false, "Enables /debug/pprof endpoints on the listen address")
 	pflag.StringVar(&listenAddress, "listen-address", ":8081", "An address and port to listen on")
 	pflag.BoolVar(&enableNodeWatch, "enable-node-watch", false, "Enables the fdb-kubernetes-monitor to watch the node resource where the current Pod is running. This can be used to read node labels")
+	pflag.StringVar(&certFile, "cert-file", "", "The location of a PEM cert for the prometheus HTTP server")
```

Comment:
I think for now this is fine, but we have to allow the fdbkubernetesmonitor to read those settings from env variables too. Right now we don't have a way to add any additional args to the fdbkubernetesmonitor, using env variables is probably the simplest approach.